packages/vue-jsoo/vue-jsoo.0.2/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Binding of Vue_js"
+maintainer: ["maxime.levillain@origin-labs.com"]
+authors: ["Maxime Levillain"]
+license: "MIT"
+homepage: "https://gitlab.com/o-labs/vue-jsoo"
+doc: "https://o-labs.gitlab.io/vue-jsoo/vue-jsoo"
+bug-reports: "https://gitlab.com/o-labs/vue-jsoo/-/issues"
+depends: [
```

Comment:
```suggestion
depends: [
  "ocaml" {>= "4.05"}
```

Some list functions that are used in the project were introduced only in more recent versions of the compiler